pkg/operator/controller/names.go

```diff
@@ -140,3 +146,11 @@ func NodeResolverDaemonSetPodSelector() *metav1.LabelSelector {
 		},
 	}
 }
+
+// FeatureGateClusterConfigName returns the namespaced name of the
+// featuregates.config.openshift.io resource of the cluster.
+func FeatureGateClusterConfigName() types.NamespacedName {
+	return types.NamespacedName{
+		Name: DefaultFeatureGate,
+	}
+}
```

Comment:
I cannot see where this function is used.

Reply:
This is not used in this PR. I'll remove the function.